libs/langchain/langchain/llms/huggingface_text_gen_inference.py

```diff
@@ -80,13 +80,16 @@ class HuggingFaceTextGenInference(LLM):
     typical_p: Optional[float] = 0.95
     temperature: float = 0.8
     repetition_penalty: Optional[float] = None
+    return_full_text: Optional[bool] = False
     truncate: Optional[int] = None
     stop_sequences: List[str] = Field(default_factory=list)
     seed: Optional[int] = None
     inference_server_url: str = ""
     timeout: int = 120
     server_kwargs: Dict[str, Any] = Field(default_factory=dict)
     streaming: bool = False
+    do_sample: Optional[bool] = False
```

Comment:
can all of these just be bools (don't need Optional)?

Reply:
My bad, `Optional` is definitely not needed for bool types. Thanks for fixing it